### PR TITLE
Completed Basic setup and home page

### PR DESCRIPTION
**Pull request by varun1233**

### CODE REVIEW

**Review by mehulsatardekar — APPROVED** [review]
yeah didint find any issues you can merge this branch

File: pages/home/home.css [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+.category-name {
+  color: var(--primary-font-color);
+  font-size: 1.5rem;
+  line-height: 2rem;
+  padding: 0;
+}
+
+.featured-product-container {
+  padding-left: 5rem;
+  padding-right: 5rem;
```
Comment (vanshitamoolchandani) [inline pages/home/home.css]: Change- You can use shorthand property like
padding: 0 5rem;

File: pages/home/home.css [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+  flex-wrap: wrap;
+  gap: 5px;
+}
+.card-vertical {
+  cursor: pointer;
+}
+
+.services-container {
+  padding-left: 5rem;
+  padding-right: 5rem;
```
Comment (vanshitamoolchandani) [inline pages/home/home.css]: Same here.

**Review by vanshitamoolchandani — APPROVED** [review]
Just one small change that I mentioned. Other than that you can merge.

File: pages/home/home.css (hunk 1 above)
Comment (varun1233) [reply pages/home/home.css]: Done

File: pages/home/home.css (hunk 2 above)
Comment (varun1233) [reply pages/home/home.css]: Done

**Pull request merged**